Assets/GothicVR/Scripts/World/DijkstraWayPointCreator.cs

```diff
@@ -0,0 +1,54 @@
+using GVR.Extensions;
+using GVR.Phoenix.Data;
+using PxCs.Data.WayNet;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace GVR
+{
+    public class DijkstraWayPointCreator : MonoBehaviour
+    {
+        /// <summary>
+        /// A dictionary containing all the DijkstraWaypoints in the scene, with their names as keys.
+        /// </summary>
+        public static Dictionary<string, DijkstraWaypoint> DijkstraWaypoints;
+
+        public static Dictionary<string, DijkstraWaypoint> Create(WorldData world)
```

Comment:
Please add some comments what you're doing here and why.

Reply:
adressed

Reply:
added comments